src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -1015,10 +1025,10 @@ class AdjudicationsService(
 
   private inline fun <reified T> String.fromJson(): T = objectMapper.readValue(this)
 
-  private suspend fun PunishmentDto.toNomisAward() = HearingResultAwardRequest(
+  private suspend fun PunishmentDto.toNomisAward(hearingDate: LocalDate? = null) = HearingResultAwardRequest(
     sanctionType = this.type.toNomisSanctionType(),
     sanctionStatus = this.toNomisSanctionStatus(),
-    effectiveDate = this.schedule.startDate ?: this.schedule.suspendedUntil ?: LocalDate.now(),
+    effectiveDate = this.schedule.startDate ?: this.schedule.suspendedUntil ?: hearingDate ?: LocalDate.now(),
     sanctionDays = this.schedule.takeIf { it.measurement == PunishmentScheduleDto.Measurement.DAYS }?.duration ?: 0,
```

Comment:
only use the hearing date if DPS does not provide a schedule (e.g not suspended or has a real start date)